README.md

```diff
@@ -10,6 +10,8 @@ The `efs-utils` package has been verified against the following Linux distributi
 | ------------ | ------------ | ------------- |
 | Amazon Linux 2017.09 | `rpm` | `upstart` |
 | Amazon Linux 2 | `rpm` | `systemd` |
+| Almalinux 8.4 | `rpm` | `systemd` |
+| RockyLinux 8.4 | `rpm` | `systemd` |
```

Comment:
We will add the distribution in README once we finish certain validation on our side. For now we can perform the rest of the change, so please revert the change in README for now, thanks.

Reply:
Reverted.